core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/OrderDAO.java

```diff
@@ -0,0 +1,131 @@
+package ca.jrvs.apps.jdbc;
+
+import ca.jrvs.apps.jdbc.util.DataAccessObject;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class OrderDAO extends DataAccessObject<Order> {
+
+  //SQL queries for functions.
+  private final static String GET_BY_ID = "SELECT c.first_name, c.last_name, c.email, o.order_id, o.creation_date, " +
```

Comment:
I know final static and static final are the same, but I still suggest you to stay the same.